components/DetailCountry/index.js

```diff
@@ -1,5 +1,5 @@
 import { styled } from "styled-components";
```

Comment:
```suggestion
import styled  from "styled-components";
```
the styled from styled components ist not destructured, otherwise you will have an errormessage in the console :)